src/index.js

```diff
@@ -1,10 +1,42 @@
-function component() {
-  const element = document.createElement('div');
+import './modules/style.css';
 
-  // Lodash, currently included via a script, is required for this line to work
-  // element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+function showToDo(todo) {
+  const list = document.querySelector('.list');
+  const newlist = document.createElement('li');
+  newlist.classList.add('newlist');
+  newlist.innerHTML = `
+    <input id="${todo.index}" class="check" type="checkbox"/>
+    <span class="text">${todo.description}</span>
```

Comment:
Your code for rendering the tasks dynamically looks great. However, you seem to not mark the checkbox based on whether the task is completed or not. 

Something like this can fix it:
`<input type="checkbox" ${data.completed ? 'checked' : ''} class="check"/>`

Reply:
I fixed it.